token-service-client/cmr/client/SslClient.java

```diff
@@ -0,0 +1,144 @@
+package cmr.client;
+
+import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLSocketFactory;
+
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+import java.util.List;
+
+public class SslClient {
+
+	private static final String LINE_BREAKER = System.getProperty("line.separator");
+	// Launchpad Token Service root url, this is set to the Sandbox
+	private static final String LAUNCHPAD_TOKEN_SERVICE_URL_ROOT = "https://api.launchpad-sbx.nasa.gov/";
+	// full path of cacerts which has the PKI certificate for connecting to token service
+	private static final String CACERTS_PATH = "/Users/yliu10/cacerts";
```

Comment:
Nitpick suggestion
```suggestion
	private static final String CACERTS_PATH = "~/cacerts";
```

Reply:
I don't like relative path. I can change it to `/path-to-cacerts/cacerts`.